app/models/exercise/approach/tag.rb

```diff
@@ -0,0 +1,10 @@
+class Exercise::Approach::Tag < ApplicationRecord
```

Comment:
What's the advantage of us having this as a many-to-many table, rather than materialising this as a JSON field on the approach? As later on, we just reget all the data into one object then work on it, I'm not sure what we're gaining from this.

Reply:
I used a many-to-many table as I originally intended some fancy queries, which would not have worked as easily with a JSON field. But maybe a JSON field is easier.